man1/aur-fetch.1

```diff
@@ -0,0 +1,43 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch \- fetches packages from a location
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.OP -L log_dir
+.OP -m mode
+.YS
+
+.SH DESCRIPTION
+\fIaur-fetch\fR accepts package names from stdin, and attempts to
+download them. It understands two \fImodes\fR of operation; \fIgit\fR
+and \fItar\fR.
+
+\fIgit\fR attempts to download the package from a git repository.
+
+\fItar\fR attempts to downloade the tar archive with
+\fIaur-fetch-snapshot\fR.
```

Comment:
Implementation detail, probably better mentioned in the `-t` option description (`.SH OPTIONS`)